<!DOCTYPE html>
<html lang="en-us">
  <head>
    <meta charset="UTF-8">
    <title>Marsala by xielab</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="stylesheets/github-light.css" media="screen">
  </head>
  <body>
    <section class="page-header">
      <h1 class="project-name">Marsala</h1>
      <h2 class="project-tagline">For better future of people</h2>
      <a href="https://github.com/xielab/marsala" class="btn">View on GitHub</a>
      <a href="https://github.com/xielab/marsala/zipball/master" class="btn">Download .zip</a>
      <a href="https://github.com/xielab/marsala/tarball/master" class="btn">Download .tar.gz</a>
    </section>

    <section class="main-content">
      <h3>
<a id="what-is-marsala" class="anchor" href="#what-is-marsala" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>What is Marsala?</h3>

<p>Marsala is a program that comprehensively detects variants (SNV and CNV) in PGS with linkage analysis.<br>
Marsala runs on Linux.</p>

<h3>
<a id="what-types-of-input-can-i-use-with-marsala" class="anchor" href="#what-types-of-input-can-i-use-with-marsala" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>What types of input can I use with Marsala?</h3>

<p>Marsala is designed to work with reads from whole genome sequencing, although users have been successful in using Marsala with reads from specific region of genome.<br>
In general, Marsala uses sequencing information from three parts:<br>
1. Disease-Grandparent or Disease-child (A)<br>
2. Disease-parent (F)<br>
3. Embryo (E)<br>
In addition, users can use sequencing information from polar body 1 (PB1) and polar body 2 (BP2) that corresponds to specific embryo, which will make result more accurate.</p>

<h3>
<a id="how-does-marsala-detect-variants" class="anchor" href="#how-does-marsala-detect-variants" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>How does Marsala detect variants?</h3>

<ol>
<li><p>SNV <br>
First, given specific genomic region (less than 2Mbp) Marsala detects SNVs for A, F and E (as well as PB1 and PB2) seperately. <br>
Then, Marsala compares the genotypes between A amd F and infer which one of the chromosome homologous was passed from A to F. (i.e. risk genomic mate determination) <br>
Next, Marsala compares the genotypes between F and E and estimate the ratio of risk passed sites to all passed sites. <br>
Finally, Marsala determines whether this E obtains the risk genomic region which contains disease-related mutation. <br></p></li>
<li><p>CNV <br>
Marsala estimate the DNA copy number of E and report whether E was normal.<br>
This protocol has been validated in following research: <br><br>
Liying Yan,  Lei Huang,  Liya Xu,  Jin Huang,  Fei Ma,  Xiaohui Zhu,  Yaqiong Tang,  Mingshan Liu, Ying Lian,  Ping Liu,  Rong Li,  Sijia Lu,  Fuchou Tang,  Jie Qiao,   X. Sunney Xie.  Live births after simultaneous avoidance of monogenic diseases and chromosome abnormality by next-generation sequencing with linkage analyses.PNAS 2015 112 (52) 15964-15969</p></li>
</ol>

<h3>
<a id="authors-and-contributors" class="anchor" href="#authors-and-contributors" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Authors and Contributors</h3>

<p>This program was built by Feng Tian and LuoXing Xiong and supported by Lei Huang and Xie Sunney.  </p>

<h3>
<a id="support-or-contact" class="anchor" href="#support-or-contact" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Support or Contact</h3>

<p>Having trouble with Marsala? Check out our <a href="http://xielab.github.io/marsala/documentation">documentation</a>.<br>
Also you can communicate with Feng (<a href="https://github.com/tianfree" class="user-mention">@tianfree</a>) and LuoXing (<a href="https://github.com/XiongLuoxing" class="user-mention">@XiongLuoxing</a>) for further help.</p>

      <footer class="site-footer">
        <span class="site-footer-owner"><a href="https://github.com/xielab/marsala">Marsala</a> is maintained by <a href="https://github.com/xielab">xielab</a>.</span>

        <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
      </footer>

    </section>

  
  </body>
</html>
